sannytee/flight-booking-api
# #166104180: implement user signup feature and testing

#### What does this PR do?
This PR set up creating an account feature and testing

### How should this be manually tested?
- Pull the branch to your local machine

- check the env.sample file for required environment variables and source them.

- start the server

- visit the route `/api/v1/auth/signup/` with the neccesary details to create an account

- run `python manage.py test` to run the test


#### What are the relevant pivotal tracker stories?
[#166104180](https://www.pivotaltracker.com/story/show/166104180)
